### PR TITLE
travel to Berlin 1989

### PR DESCRIPTION
**Pull request by YIFANNIU**

### CODE REVIEW

**Comment by YIFANNIU** [comment]
please review ,@casey-collab

File: english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+You suddenly travel to Berlin 1989, lying on the road, at night (without wearing cloth!!).
```
Comment (casey-collab) [inline english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md (hunk 1 above)
Comment (YIFANNIU) [inline english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md]: i continue with other people's one option.... so i leave it for other people adding option under my .md file

Comment (cbuckey-uda) [inline english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md]: Casey is not saying that this file needs a link in it. Rather, the rest of the story needs a link that leads to this file. If a reader starts by choosing their language in `language.md`, then keeps clicking links, you want it to be possible for them to reach this file.

Comment (YIFANNIU) [inline english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md]: I add a link in that person's .md file so reader can choose "travel to Berlin" which link to my .md file

Comment (YIFANNIU) [inline english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md]: You can check follow this flow: english/light-fire/xyzzy/stand-petrified/meet-wizard.md(i continue from this .md) , i add a link in his .md file that reader can jump to my .md file(path is : english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md)

Comment (cbuckey-uda) [inline english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md]: If you look at the "files changed" tab on your pull request, I believe you will not see that change showing up. Did you forget to commit the change, or forget to push it?

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**